src/operators/only-instances-of.ts

```diff
@@ -18,8 +18,14 @@ export function instanceOf<A>(clazz: Newable<A>) {
  *
  * @param clazz which instances are passed through
  */
-export function justInstanceOf<A>(clazz: Newable<A>) {
+export function onlyInstancesOf<A>(clazz: Newable<A>) {
```

Comment:
Should be onlyInstanceOf in my opinion. All other operators also use singular wording. It is skipNull skipNulls.